junebug/api.py

```diff
@@ -172,13 +177,16 @@ def delete_channel(self, request, channel_id):
             'type': 'object',
             'properties': {
                 'to': {'type': 'string'},
-                'from': {'type': 'string'},
+                'from': {'type': ['string', 'null']},
                 'reply_to': {'type': 'string'},
+                'content': {'type': ['string', 'null']},
                 'event_url': {'type': 'string'},
                 'priority': {'type': 'string'},
                 'channel_data': {'type': 'object'},
-            }
+            },
+            'required': ['from', 'content'],
```

Comment:
Do we want to add validation for people giving fields they shouldn't be giving, for e.g. `transport_name`. People (and machines) might find it surprising that the values they gave for fields were silently overridden with our own fields.